src/Forms/Prism.Forms/Navigation/NavigationException.cs

```diff
@@ -12,6 +12,7 @@ public class NavigationException : Exception
         public const string IConfirmNavigationReturnedFalse = "IConfirmNavigation returned false";
         public const string NoPageIsRegistered = "No Page has been registered with the provided key";
         public const string ErrorCreatingPage = "An error occurred while resolving the page. This is most likely the result of invalid XAML or other type initialization exception";
+        public const string AmbiguousSelectedTab = "The requested selected tab matches more than one child page";
```

Comment:
this case doesn't appear to have actually been handled. please be sure to update the appropriate location in the NavigationService